pyproject.toml

```diff
@@ -1,6 +1,7 @@
 [build-system]
 requires = [
     "setuptools >= 46.4.0",
+    "pkgconfig",
```

Comment:
Can we keep this list sorted? Is there a minimum version we'd need?

Reply:
Sure. My idea was to keep `setuptools` first since it's the backend (i.e. the primary dependency), but I can change order, no problem.

I don't think we do need a minimal version. The two functions we're using are available since the first release, 0.1.0. I've just tried forcing `== 0.1.0` and the build succeeded.

Reply:
Ah, makes sense. I often make this separation by having an empty line and adding code comments marking essential and plugin sections of the list. But since it wasn't here, I forgot about that habit :)